### ISSUE TITLE
Package can't be imported with module type projects

### ISSUE THREAD

**Issue by vchirikov** [only-jsx/jsx-runtime#17]
Hi, I want to use your package to render compiled mdx, so I tried to use it like this:
```
import * as jsxRuntime from 'only-jsx/jsx-runtime';
const result = new Function(code)({ jsxRuntime });
```
But I got `[vite] Internal server error: Failed to resolve import "only-jsx/jsx-runtime"` error, although I don't see errors with vs code and can navigate to the type definitions. It's in an ESM project (`"type": "module",` in package.json). So I think it could be something wrong with ESM support in the package. Can you check this or maybe you can provide an example with esm project?

**Comment by only-jsx** [comment]
Hi,

I have checked the issue.

Yes, there were error in the package.json exports property.

I think that I fixed it now and published new version 0.1.1.

You can use only-jsx library for rendering mdx after ```npm i only-jsx```.

## Examples

```jsx
import {evaluateSync} from '@mdx-js/mdx'
import * as runtime from 'only-jsx/jsx-runtime'

const code = `export const Thing = () => <>World</>

# Hello <Thing />`;

const {default: Content} = evaluateSync(code, runtime);

const e = <Content/>;
```

with webpack:

```js
rules: [
    ...
    {
        test: /.mdx?$/,
        use: [
            {
                loader: '@mdx-js/loader',
                /** @type {import('@mdx-js/loader').Options} */
                options: { jsxImportSource: 'only-jsx'}
            }
        ]
    },
]
```
and then in .jsx
```jsx
import Test from './test.mdx'
```

as a standalone .js

```js
import { compileSync } from '@mdx-js/mdx'

const code = `export const Thing = () => <>World</>

# Hello <Thing />`;
const js = String(compileSync(code, { jsxImportSource: 'only-jsx' }));
console.log(js);
```

**Comment by vchirikov** [comment]
> I think that I fixed it now

Yep thanks! I can confirm now it works with ESM modules. 
Btw you can add `jsx as jsxDEV` to your exports,
